src/llmcompressor/transformers/finetune/data/custom.py

```diff
@@ -36,76 +27,4 @@ class CustomDataset(TextGenerationDataset):
 
     """
 
-    def __init__(self, data_args, split, processor):
```

Comment:
why are these removed?

Reply:
They've been moved to the base `TextGenerationDataset ` class.